packages/hub/src/squiggle/components/ImportTooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import clsx from "clsx";
+import { FC } from "react";
+import { graphql, useLazyLoadQuery } from "react-relay";
+
+import { ModelCard } from "@/models/components/ModelCard";
+
+import { parseSourceId } from "./linker";
+
+import { ImportTooltipQuery } from "@/__generated__/ImportTooltipQuery.graphql";
+
+type Props = {
+  importId: string;
+};
+
+export const ImportTooltip: FC<Props> = ({ importId }) => {
+  const { owner, slug } = parseSourceId(importId);
+
+  const { model } = useLazyLoadQuery<ImportTooltipQuery>(
+    graphql`
+      query ImportTooltipQuery($input: QueryModelInput!) {
+        model(input: $input) {
+          __typename
+          ... on Model {
+            ...ModelCard
+          }
+        }
+      }
+    `,
+    { input: { owner, slug } }
+  );
+
+  if (model.__typename !== "Model") {
+    return (
+      <div>
+        {"Couldn't load "}
+        {owner}/{slug}
```

Comment:
Why not  use interpolation here       
```
{`Couldn't load ${owner}/${slug}\`}
```

Reply:
I like React style a bit more (no `$`), but you're right that mixing both styles here is too much